Payment/Expense/index.spec.ts

```diff
@@ -4,10 +4,12 @@ describe("Payment.Expense", () => {
 	const expense: issuefab.Payment.Expense = {
 		type: "expense",
 		limit: [10, "EUR"],
-		paid: {},
+		paid: { abcd0001: { issuer: "tobe@kakal.cse", created: "2021-12-20T13:37:42Z", modified: "2021-12-20T13:37:42Z" } },
```

Comment:
do we want to create an additional test where the timestamps are not the same?

Reply:
We could, but we don't check that anywhere. Today, we don't use Created at all, and modified is only tested in the API.